scripts/updateRecordsCatalogLink.py

```diff
@@ -0,0 +1,43 @@
+import os
+import re
+
+from model import Record
+from main import loadEnvFile
+from managers import DBManager
+
+
+def main():
+    '''
+    Updating NYPL catalog records with new link to replace text/html link
+    '''
+
+    loadEnvFile('local-qa', fileString='config/{}.yaml')
+
+    dbManager = DBManager(
+        user= os.environ.get('POSTGRES_USER', None),
+        pswd= os.environ.get('POSTGRES_PSWD', None),
+        host= os.environ.get('POSTGRES_HOST', None),
+        port= os.environ.get('POSTGRES_PORT', None),
+        db= os.environ.get('POSTGRES_NAME', None)
+    )
+
+    dbManager.generateEngine()
+
+    dbManager.createSession()
+
+    catalogRegex = r'catalog: true'
+    linkRegex = r'text/html'
+
+    for record in dbManager.session.query(Record) \
+        .filter(Record.source == 'nypl').all():
+            
+            for i in record.has_part:
+                if re.search(catalogRegex, i) != None:
+                    if re.search(linkRegex, i) != None:
+                        i.replace('text/html', 'application/html/catalog')
```

Comment:
I'm not 100% sure this will persist the change in the database. We might need to update the full `has_part` array, should try to evaluate that in QA with a limited test before merging this